src/viam/sdk/tests/test_servo.cpp

```diff
@@ -0,0 +1,153 @@
+#define BOOST_TEST_MODULE test module test_servo
+
+#include <typeinfo>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include <boost/test/included/unit_test.hpp>
+
+#include <viam/api/common/v1/common.pb.h>
+#include <viam/api/component/servo/v1/servo.grpc.pb.h>
+#include <viam/api/component/servo/v1/servo.pb.h>
+
+#include <viam/sdk/common/proto_type.hpp>
+#include <viam/sdk/components/servo/client.hpp>
+#include <viam/sdk/components/servo/server.hpp>
+#include <viam/sdk/components/servo/servo.hpp>
+#include <viam/sdk/tests/mocks/mock_servo.hpp>
+#include <viam/sdk/tests/test_utils.hpp>
+
+BOOST_TEST_DONT_PRINT_LOG_VALUE(std::vector<viam::sdk::GeometryConfig>)
+
+namespace viam {
+namespace sdktests {
+namespace servo {
+
+using namespace viam::sdk;
+
+BOOST_AUTO_TEST_SUITE(test_mock)
+
+BOOST_AUTO_TEST_CASE(mock_get_position) {
+    std::shared_ptr<MockServo> servo = MockServo::get_mock_servo();
+    servo->move(static_cast<unsigned>(3));
```

Comment:
What's up with these `static_cast`s? Tests won't compile without them? Unfortunate that users will have to do this, but maybe that's a normal C++ thing, @stuqdog?

Reply:
Oh, interesting. Before it was complaining about types so I made these changes so that it would be of type `uint32_t`, but now it's working without doing so. Will be removing these now

Reply:
Ah nice! 🎉 . I would be surprised if C++ cannot implicitly convert `3` to a `uint32_t`.